pkgs/development/python-modules/frida-python/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkgs,
+  typing-extensions,
+}: let
+  pname = "frida-python";
+  version = "16.0.19";
+  namePypi = "frida";
+
+  devkit = fetchurl {
+    url = "https://github.com/frida/frida/releases/download/${version}/frida-core-devkit-${version}-linux-${stdenv.hostPlatform.linuxArch}.tar.xz";
+    hash = "sha256-yNXNqv8eCbpdQKFShpAh6rUCEuItrOSNNLOjESimPdk=";
+  };
+in
+  pkgs.python3Packages.buildPythonPackage rec {
```

Comment:
why are you using `pkgs` in python-modules, please read the expressions of other modules

Reply:
woops. Fixed.